packages/extension-base/src/koni/api/staking/paraChain.ts

```diff
@@ -21,8 +21,17 @@ function getSingleStakingAmplitude (substrateApi: _SubstrateApi, address: string
     [substrateApi.api.query.parachainStaking.delegatorState, address],
     [substrateApi.api.query.parachainStaking.unstaking, address]
   ], async ([_delegatorState, _unstaking]) => {
-    const delegatorState = _delegatorState.toPrimitive() as unknown as ParachainStakingStakeOption;
-    const unstakingInfo = _unstaking.toPrimitive() as unknown as Record<string, number>;
+    let delegatorState: ParachainStakingStakeOption;
+
+    if (_STAKING_CHAIN_GROUP.krest_network.includes(chain)) {
+      const krestDelegatorState = _delegatorState.toPrimitive() as unknown as KrestDelegateState;
+
+      delegatorState = krestDelegatorState?.delegations[0] as unknown as ParachainStakingStakeOption;
```

Comment:
It's better to iterate through all delegation record than assuming there's only 1 record

Reply:
amplitude and krest are currently using the same function to subscribe nominator metadata; this handling code also forces the amplitude nominator, which is always 1 record, to be an array of one record